MFVNeutralino/plugins/GenParticles.cc

```diff
@@ -784,36 +862,40 @@ void MFVGenParticles::produce(edm::Event& event, const edm::EventSetup&) {
       // If there is a neutralino or stop in the first event, assume that's the
       // LSP id wanted. Otherwise, default to looking for gluino. This
       // isn't relevant for some signals.
-      lsp_id = 1000021;
-      for (int i = 0, ie = int(gen_particles->size()); i < ie; ++i)
-        if      (gen_particles->at(i).pdgId() == 1000022) { lsp_id = 1000022; break; }
+      lsp_id = 1000006;
+      for (int i = 0, ie = int(gen_particles->size()); i < ie; ++i) {
+	      if      (gen_particles->at(i).pdgId() == 1000022) { lsp_id = 1000022; break; }
         else if (gen_particles->at(i).pdgId() == 1000006) { lsp_id = 1000006; break; }
         else if (gen_particles->at(i).pdgId() == 9000006) { lsp_id = 9000006; break; }
+      }
```

Comment:
Is this change needed? @abbywarden I think this one is you? I would expect to just reach the `else if` line and that it would be set as needed for stop -> ld / lb.